src/components/GridComponent.js

```diff
@@ -104,7 +120,7 @@ class PathfinderGrid extends Component {
                                   type = 'end';
                                 }
                                 let fogClassName;
-                                if (visibleCells && !visibleCells.has(cell)) {
+                                if (visibleCells && !visibleCells.visibleCells.has(cell)) {
```

Comment:
as i've said before, if you have to do `namedVariable.namedVariable` at any time, you better have darn good reason

and i don't see one here

Reply:
that is one option, yes
another option is – if it's two things, why not have two things, instead of one thing that holds two things?

Reply:
cause we needed both in one return value, unless you think I should pass in another empty holder like seenCells

Reply:
🐇